examples/wallets/src/lib.rs

```diff
@@ -189,6 +189,43 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    #[allow(unused_variables)]
+    async fn setup_wallet_custom_assets() -> Result<(), Box<dyn std::error::Error>> {
+        // ANCHOR: custom_assets_wallet
+        use fuels::prelude::*;
+        use rand::Fill;
+
+        let mut wallet = LocalWallet::new_random(None);
+        let mut rng = rand::thread_rng();
+
+        let asset_id_base = BASE_ASSET_ID;
+        let coins_per_asset_base = 2;
+        let amount_per_coin_base = 4;
+
+        let mut asset_id_1 = AssetId::zeroed();
+        asset_id_1.try_fill(&mut rng)?;
+        let coins_per_asset_1 = 6;
+        let amount_per_coin_1 = 8;
+
+        let mut asset_id_2 = AssetId::zeroed();
+        asset_id_2.try_fill(&mut rng)?;
+        let coins_per_asset_2 = 10;
+        let amount_per_coin_2 = 12;
+
+        let assets = &[
+            (asset_id_base, coins_per_asset_base, amount_per_coin_base),
+            (asset_id_1, coins_per_asset_1, amount_per_coin_1),
+            (asset_id_2, coins_per_asset_2, amount_per_coin_2),
+        ];
```

Comment:
I feel that using `Vec` here would be cleaner. Also, maybe a struct would be in order here? Something like `AssetSetup` with the 3 fields?

Reply:
This was definitely on my mind as you need to know the order. However the requested function was a vector with tuples. If we agree on the change I will gladly implement it.

Reply:
Ah you're right, let's ping @digorithm to see what he thinks of a struct. the struct could be built using helper functions taki